### DISCUSSION TITLE
"Are you sure" when hitting Esc while in edit mode with an active more-info box

### DISCUSSION THREAD

**Discussion by BoneheadFraggle** [home-assistant/frontend#7504] [Dashboard] [answered]
Background:
If you click an entity while editing a card in the GUI, the more-info dialog shows. 
If you then press Esc to get rid of the more-info box, the editor closes without any confirmation and without saving. 

I would prefer it closed the more-info and got me back to the card editor - or at least gave me an "Are you sure" before closing down the editor.

**Comment by matthiasdebaat** [comment, marked as answer]
Yes, that seems like a bug to me. Is this still in the current HA version?

**Reply by BoneheadFraggle** [reply]
Yep, just tried it.
Version | core-2021.8.8
Installation Type | Home Assistant OS

To reproduce:

1. Edit Dashboard
2. Edit any card
3. Click an entity on that card
4. When the more-info shows, press Escape
5. Boom, the card immediately close and you're back on the Dashboard

**Reply by matthiasdebaat** [reply]
Thanks, just made this issue: https://github.com/home-assistant/frontend/issues/9881
